staging/src/k8s.io/kubectl/pkg/cmd/cmd_test.go

```diff
@@ -236,19 +250,32 @@ func TestKubectlCommandHandlesPlugins(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			pluginsHandler := &testPluginHandler{
 				pluginsDirectory: "plugin/testdata",
+				validPrefixes:    plugin.ValidPluginFilenamePrefixes,
 			}
 			ioStreams, _, _, _ := genericiooptions.NewTestIOStreams()
-
 			root := NewDefaultKubectlCommandWithArgs(KubectlOptions{PluginHandler: pluginsHandler, Arguments: test.args, IOStreams: ioStreams})
-			if err := root.Execute(); err != nil {
-				t.Fatalf("unexpected error: %v", err)
+			if pluginsHandler.lookedup || pluginsHandler.executed {
```

Comment:
We haven't run `root.Execute` yet, how `lookedup` or `executed` can be true?

Reply:
because DefaultPluginHandler is implemented by exec call as commented bellow, so `root.Execute` will never be called in the end